docs/source/api/python/transform.rst

```diff
@@ -5,8 +5,6 @@ dgl.transform
 
 .. automodule:: dgl.transform
 
-Common algorithms on graphs.
```

Comment:
Common algorithms for transforming graph structures?

Reply:
This file is removed and merged to dgl.rst.